apple/bundling/binary_support.bzl

```diff
@@ -27,7 +27,7 @@ def _create_swift_runtime_linkopts_target(
         name,
         deps,
         is_static,
-        tags = None,
+        tags,
```

Comment:
Lets be consistent and drop the default on `test_only` also then, it likely will help ensure it properly gets relayed if something else tries to use this.

Reply:
Done